packages/runtime/lib/worker/main.js

```diff
@@ -94,7 +94,7 @@ async function main () {
   globalThis.fetch = fetch
   globalThis[kId] = threadId
   globalThis.platformatic = Object.assign(globalThis.platformatic ?? {}, {
-    logger: createLogger(),
+    logger: globalThis.platformatic?.logger ?? createLogger(globalThis.platformatic?.logLevel),
```

Comment:
why is this needed? Can you add a comment?

Reply:
I've reverted this